helpdesk_fieldservice/models/helpdesk_ticket.py

```diff
@@ -78,3 +81,14 @@ def action_create_order(self):
         res = self.env.ref('fieldservice.fsm_order_form', False)
         result['views'] = [(res and res.id or False, 'form')]
         return result
+
+    @api.depends('fsm_order_ids', 'stage_id', 'fsm_order_ids.stage_id')
+    def _compute_all_closed(self):
+        for ticket in self:
+            ticket.all_orders_closed = True
+            if ticket.fsm_order_ids:
+                for order in ticket.fsm_order_ids:
+                    if order.stage_id.name not in  ['Closed', 'Cancelled']:
+                        ticket.all_orders_closed = False
+            else:
+                ticket.all_orders_closed = False                
```

Comment:
```suggestion
                ticket.all_orders_closed = False
```

Reply:
@osi-scampbell Do we need the `else` statement? If the ticket does not have FSM orders, then they are all closed...